lib/MusicBrainz/Server/Edit/Release/Merge.pm

```diff
@@ -219,6 +219,11 @@ override build_display_data => sub
                 mediums => $_->{mediums}
             }, @{ $self->data->{medium_changes} }
         ];
+        $data->{empty_releases} = [
+            map +{
+                release => $loaded->{Release}{ $_->{id} } // Release->new(name => $_->{name}),
+            }, grep { scalar @{ $_->{mediums} } == 0 } @{ $self->data->{old_entities} }
```

Comment:
The `mediums` array was added in https://github.com/metabrainz/musicbrainz-server/commit/b630292550f7b44af64e48a4ec27ea7922b749a7 (marked as `Optional`), so not all edits will have it.

`@{ $_->{mediums} }` will throw an error where `$_->{mediums}` is undef, so we'll have to check that it's defined first:
```suggestion
            }, grep { defined $_->{mediums} && scalar @{ $_->{mediums} } == 0 } @{ $self->data->{old_entities} }
```